### PR TITLE
Add Infrastructure As Code (Iac) advance scan

### PR DESCRIPTION
**Pull request by attiasas**
- [x] All [tests](https://github.com/jfrog/jfrog-vscode-extension#building-and-testing-the-sources) passed. If this feature is not already covered by the tests, I added new tests.
- [x] I used `npm run format` for formatting the code before submitting the pull request.
-----

depend on https://github.com/jfrog/jfrog-vscode-extension/pull/333

### CODE REVIEW

**Review by yahavi — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ To learn more, see [here](https://www.jfrog.com/confluence/display/JFROG/Vulnera
 |---------------------------------------------------|:---:|:-----:|:------:|:---:|:-------:|:------:|
 | SCA                                               |  ✅  |   ✅   |   ✅    |  ✅  |    ✅    |   ✅    |
 | Upgrade vulnerable dependencies to fixed versions |  ✅  |   ✅   |   ✅    |  ✅  |    ✅    |   ✅    |
+| Infrastructure As Code (Iac)                               |  ✅  |   ✅   |   ✅    |  ✅  |    ✅    |   ✅    |
```
[the remark this answers is not in the document]
Comment (attiasas) [reply README.md]: removed from doc until official release

[the remark this answers is not in the document]
Comment (attiasas) [reply README.md]: removed from doc until official release

File: src/main/treeDataProviders/utils/analyzerUtils.ts [hunk 2]
```diff
@@ -27,7 +46,9 @@ export class AnalyzerUtils {
      */
     public static parseLocationFilePath(filePath: string): string {
```
[the remark this answers is not in the document]
Comment (attiasas) [reply src/main/treeDataProviders/utils/analyzerUtils.ts:51]: this is not a bug, this is how the SARIF format works, it's not supposed to depend on the platform the tool is running on.
I added explanation and changed the method to be more efficient

File: src/main/treeDataProviders/utils/analyzerUtils.ts [hunk 3]
```diff
@@ -246,6 +267,63 @@ export class AnalyzerUtils {
...
+                    ' seconds)',
+                'INFO'
+            );
+            root.apply();
+        }
+        progressManager.reportProgress();
+    }
+
+    /**
+     * Populate Iac information in
```
[the remark this answers is not in the document]
Comment (attiasas) [reply src/main/treeDataProviders/utils/analyzerUtils.ts]: nice, added the missing doc

**Pull request merged**